03-Running AzTS solution from UI/README.md

```diff
@@ -70,6 +71,27 @@ Here are some additional video tutorials explaining the features provided by AzT
 ### **How to export control scan logs to local machine**
 [![UI](../Images/04_UI_ExportToCSV.gif)](https://azsk-azts-cdn.azureedge.net/videosforpublicgithubdoc/Ext_Export_To_CSV.mp4)
 
+## Miscellanoues UI features
+- ### **Scan In Progress:**
+
+AzTS UI can be used to scan subscriptions(s) manually. Once such manusal scan is triggered, you can get notification about scan progress status (e.g. Scan In Progress or Scan completed) by enabling Scan In Progress feature in AzTS UI. 
+This feature is not enabled by default. If you have not enabled this feature in your AzTS setup yet, please follow steps mentioned below:
+
+- Open the [Azure portal](https://portal.azure.com/).
+- Navigate to AzTS host subscription -> AzTS host resource group.
+- Go to required AzTS API app service(AzSK-AzTS-WebApi-xxxxx).
+- Go to Configuration under Settings.
+- Click on "New application setting" to add required appsettings.
+- Add **FeatureManagement__OnDemandScanProgressIndicator** as the Name of the appsetting.
+- Add **true** as the value of the appsetting.
+- Add **UIConfigurations__OnDemandScanProgressIndicatorConfiguration__IsEnabled** as the Name of the appsetting.
+- Add **true** as the value of the appsetting.
+Save these settings. This will restart AzTS API app service.
+
+- Assign "Storage Table Data Contributor" role to Managed Identity for AzTS WebAPI(AzSK-AzTS-WebApi-xxxxx) on the Storage Account in AzTS Host Resource Group(AzSK-AzTS-Storage-xxxxx).
+
+ Next time, when you open AzTS UI and scan subscriprion(s) manually, an indicator about Scan In Progress or scan completed will be shown.
```

Comment:
We can highlight scan in progress indicator.

Reply:
NIT: Let's use same masking colors in every screenshot.

Reply:
Updated, thanks!!